docker/server.xml

```diff
@@ -40,23 +40,14 @@
     -->
      <Resource type="javax.sql.DataSource"
         name="jdbc/mifosplatform-tenants"
-        factory="org.apache.tomcat.jdbc.pool.DataSourceFactory"
+        factory="com.zaxxer.hikari.HikariJNDIFactory"
         driverClassName="${fineract_tenants_driver}"
-        url="${fineract_tenants_url}"
+        jdbcUrl="${fineract_tenants_url}"
         username="${fineract_tenants_uid}"
         password="${fineract_tenants_pwd}"
-        initialSize="3"
-        maxActive="10"
-        maxIdle="6"
-        minIdle="3"
-        validationQuery="SELECT 1"
-        testOnBorrow="true"
-        testOnReturn="true"
-        testWhileIdle="true"
-        timeBetweenEvictionRunsMillis="30000"
-        minEvictableIdleTimeMillis="60000"
-        logAbandoned="true"
-        suspectTimeout="60"
+        minimumIdle="3"
+        maximumPoolSize="10"
+        idleTimeout="60000"
       />
```

Comment:
because for FINERACT-796 we (you) are correctly and as expected replacing the `<jee:jndi-lookup ...>` by a simple regular local Spring `<bean>`, this entire JNDI Resource in Tomcat's server.xml should not be required anymore - please remove this, it should still work.